Add Cron Trigger template to Examples and clarify docs (#1959)
cloudflare/cloudflare-docs: products/workers/src/content/examples/cron-trigger.md

---
order: 1000
type: example
summary: Set a Cron Trigger for your Worker.
tags:
  - Middleware
pcx-content-type: configuration
---

# Cron Triggers

<ContentColumn>
  <p>{props.frontmatter.summary}</p>
</ContentColumn>

```js
addEventListener('scheduled', event => {
  event.waitUntil(triggerEvent(event.request))
})

async function triggerEvent(request) {
    // Fetch some data
    // Update API
  return new Response("OK")
}
    /* 
    If you're doing this in Wrangler, set the cron syntax (once per hour shown below) by adding this to your wrangler.toml:
    
    [triggers]
    crons = ["0 * * * *"] 

>     */

Reviewer: Some funky spacing going on here

Author: De-funked. Thank you

---
order: 1000
type: example
summary: Set a Cron Trigger for your Worker.
tags:
  - Middleware
pcx-content-type: configuration
---

# Cron Triggers

<ContentColumn>
  <p>{props.frontmatter.summary}</p>
</ContentColumn>

```js
addEventListener('scheduled', event => {

>     event.waitUntil(triggerEvent(event.request))

Reviewer 2: Scheduled events don't have requests or accept responses, so the example should probably do something else (likely with `event.scheduledTime` or `event.cron`)

})

async function triggerEvent(request) {
    // Fetch some data
    // Update API
    return new Response("OK")
}
```

## Setting Cron Triggers in Wrangler

> If you're deploying with Wrangler, you can set the cron syntax (once per hour shown below) by adding this to your `wrangler.toml`:

Reviewer 3: ```suggestion
If you are deploying with Wrangler, set the cron syntax (once per hour as shown below) by adding this to your `wrangler.toml` file:
```
